packages/task/src/node/task-server.ts

```diff
@@ -56,13 +60,18 @@ export class TaskServerImpl implements TaskServer, Disposable {
             toDispose.dispose();
         }
         this.toDispose.clear();
+        this.backgroundTaskStatusMap.clear();
```

Comment:
maybe we should remove the `taskId - taskStatus` from the map after the task exits?

Reply:
i already doing it in 'disposeByTaskId' method (lines 72+73)

Reply:
Yes you are right :) my bad.